groups/p2p/register_group_connected.js

```diff
@@ -87,6 +87,10 @@ module.exports = ({coordinator, count, id, lnd}, cbk) => {
               return cbk(err);
             }
 
+            if (!res.records || !res.records.length) {
```

Comment:
is there a reason this is changed?

Reply:
It is because fanout has one less step so by the time it hits this step, records are an empty object so the decode will fail and it will error out until peers connect to the coordinator. 

This shouldn't effect group open because it will always have records at this point and it will hit decode step as usual.